features/steps/cli_steps.py

```diff
@@ -310,3 +312,35 @@ def _check_image():
             sleep(0.5)
 
     assert timeout(_check_image, duration=30)
+
+
+@then("A {filepath} file should exist")
+def check_if_file_exists(context: behave.runner.Context, filepath: str):
+    """Checks if file is present and has content.
+
+    Args:
+        context (behave.runner.Context): Behave context.
+        filepath (int): A path to a file to check for existence.
```

Comment:
```suggestion
        context: Behave context.
        filepath: A path to a file to check for existence.
```
We don't really mention types in docstrings, typehints should be enough. Also less things to keep in sync (for some reason `filepath` here was an `int` rather than `str`).

Reply:
Removed the types from docstring.